Cartfile.resolved

```diff
@@ -1,20 +1,20 @@
-github "mozilla-mobile/onepassword-app-extension" "68a9d932a6373ca045984fbf4fb9a25aee43284c"
-github "DaveWoodCom/XCGLogger" "7.0.0"
+github "DaveWoodCom/XCGLogger" "7.0.1"
 github "Dev1an/A-Star" "3.0.0-beta-1"
-github "Leanplum/Leanplum-iOS-SDK" "2.7.2"
-github "SDWebImage/SDWebImage" "5.4.0"
+github "Leanplum/Leanplum-iOS-SDK" "2.8.0"
+github "SDWebImage/SDWebImage" "5.8.4"
 github "SnapKit/SnapKit" "5.0.1"
 github "SwiftyJSON/SwiftyJSON" "5.0.0"
 github "adjust/ios_sdk" "v4.18.3"
-github "apple/swift-protobuf" "1.9.0"
-github "cezheng/Fuzi" "3.1.1"
-github "getsentry/sentry-cocoa" "4.4.3"
+github "apple/swift-protobuf" "1.11.0"
+github "cezheng/Fuzi" "3.1.2"
+github "getsentry/sentry-cocoa" "4.5.0"
 github "google/EarlGrey" "935df455f3937662575c868524ddd64078539741"
 github "jrendel/SwiftKeychainWrapper" "3.4.0"
 github "kif-framework/KIF" "v3.7.8"
 github "mozilla-mobile/MappaMundi" "1d17845e4bd6077d790aca5a2b4a468f19567934"
+github "mozilla-mobile/onepassword-app-extension" "68a9d932a6373ca045984fbf4fb9a25aee43284c"
 github "mozilla-mobile/telemetry-ios" "v1.1.3"
-github "mozilla-services/shavar-prod-lists" "1f282be9bc7bf86cf4d695e2a63dbcdf2eecfb89"
-github "mozilla/application-services" "v61.0.7"
+github "mozilla-services/shavar-prod-lists" "7b34cf7f668455d61a7195f59ce1930ce84e146c"
+github "mozilla/application-services" "v61.0.12"
```

Comment:
stage only this line in the file pls. 
Actually, the swift-protobuf line (8) probably also should be staged, I think that goes along with this PR.

Reply:
Done! I don't think we changed any of the Protobuf schemas—the telemetry gets passed as JSON across the FFI—but it's easy to revert the `swift-protobuf` bump, too.